src/ui/Tooltip.scss

```diff
@@ -0,0 +1,25 @@
+    .linktext {
+        cursor: pointer;
+        color: #0595DB;
+    }
+    
+    .iFrame {
+        width: 350px;
+        height: 350px;
+    }
+    
+    .box {
+        display: none;
+        width: 100%;
+    }
+    
+    a:hover+.box,
+    .box:hover {
+        display: block;
+        position: relative;
+        z-index: 100;
+    }
+    
+    .widget {
+        width: auto;
+    }
```

Comment:
Fix indents files, starting from column 0
Each file should end with a empty line